sway-core/src/convert_parse_tree.rs

```diff
@@ -952,10 +952,12 @@ fn fn_args_to_function_parameters(
             .collect::<Result<_, _>>()?,
         FnArgs::NonStatic {
             self_token,
+            mutable_self,
             args_opt,
         } => {
             let mut function_parameters = vec![FunctionParameter {
                 name: Ident::new(self_token.span()),
+                is_mutable: matches!(mutable_self, Some(_mut_token)),
```

Comment:
`mutable_self.is_some()` is probably more readable here.  But not a necessary change.  I was looking at it thinking 'what is `_mut_token` there for?'

Reply:
Ah yes silly me.